src/main/java/io/github/thebusybiscuit/slimefun4/utils/ArmorStandUtils.java

```diff
@@ -47,13 +49,22 @@ public class ArmorStandUtils {
      * @return The spawned {@link ArmorStand}
      */
     public static @Nonnull ArmorStand spawnArmorStand(@Nonnull Location location) {
-        return location.getWorld().spawn(location, ArmorStand.class, armorStand -> {
-            armorStand.setVisible(false);
-            armorStand.setSilent(true);
-            armorStand.setMarker(true);
-            armorStand.setGravity(false);
-            armorStand.setBasePlate(false);
-            armorStand.setRemoveWhenFarAway(false);
-        });
+        // 1.19 and below don't have the consumer method so flicker exists on these versions.
+        if (Slimefun.getMinecraftVersion().isBefore(MinecraftVersion.MINECRAFT_1_20)) {
```

Comment:
I wish we had a isAtMost but we don't haha

Reply:
We have isatleast